core/client/config/environment.js

```diff
@@ -33,7 +33,7 @@ module.exports = function (environment) {
         ENV.APP.LOG_TRANSITIONS_INTERNAL = true;
         ENV.APP.LOG_VIEW_LOOKUPS = true;
         ENV['ember-cli-mirage'] = {
-            enabled: false
+            enabled: true
```

Comment:
Not strictly needed but I liked the explicitness. Can be removed if we think it's unnecessary noise.